### DISCUSSION TITLE
Welcome to PythonStudyGroups Discussions!

### DISCUSSION THREAD

**Discussion by mutanthumb** [Digital-Preservation-Coalition/PythonStudyGroups#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by WMRG1057** [comment]
Just checking in from Cheltenham Pa to make sure this works.
Hopefully everyone has installed Python..

**Reply by mutanthumb** [reply]
Hi-
Welcome to the BCC/DPC Python Study Groups!

-susan

**Comment by asegal20902** [comment]
Which version of Python should be installed? Should we use an application such as Visual Studio Code? Thanks.

**Reply by mutanthumb** [reply]
Hi-
Specific version of Python may depend on your particular study group. You should ask this question during the first Zoom session. You can you use whichever application you feel most comfortable with, IDLE, VS Code, PyCharm, a text editor, etc.
Thanks,
susan

**Comment by asegal20902** [comment]
Also I am signed into GitHub under my personal email. Should I sign in under my work email as I am doing this under the auspices of the Library of Congress?

**Reply by mutanthumb** [reply]
Hi Ariel-
It is up to you which account you use for this program. 
-susan

**Comment by asegal20902** [comment]
Hi! I'm Ariel Segal. I'm a Digital Conversion Technician at the Library of Congress. Notable collections I have helped to digitize include LC’s Hebrew manuscripts, and 19th century credit rating books. I have an MLS, specializing in Archives, and an BA and MA in History from the University of Maryland. During the pandemic, I participated in some coding events such as "Getting Started with Python Step-By-Step" from the noted digital imaging company Digital Transitions (recording available here https://www.youtube.com/watch?v=mozYgcGzjqs ) but since getting back on site, the majority of my work time is involved with the process of digitization itself and I have not had time to follow up with Python. I hope that this course framework helps me advance again.

**Reply by mutanthumb** [reply]
Hi Ariel-
Welcome to the Python Study Groups, we're glad you're here. I hope you can get going again with Python, it's easier to remember when you have a specific thing to work on. Whenever I have gaps in usage I still find myself looking things up. 

-susan

**Comment by DEijsermans** [comment]
Hello everyone! I'm Danny Eijsermans. I'm an application administrator at the Kunstmuseum in The Hague, Netherlands. I have an MA in Art History but also extensive work experience in IT, mostly web development (backend). I currently work with the collection data for the Kunstmuseum, about 190K records in the database (Axiell Collections). I have limited experience with Python per se but I am familiar enough with coding in other languages that I found it easy to quickly understand Python (with the occasional help from everyone's favorite AI, ChatGPT). My goal is to become more fluent in Python, especially data-driven modules/packages, to enable extensive research in my datasets.

**Reply by mutanthumb** [reply]
Welcome to the Python Study Groups, Danny!

Be sure to discuss your goals with your mentor.
-susan

**Comment by asegal20902** [comment]
Will we connect with/get mentors at the sessions?

**Reply by mutanthumb** [reply]
You should get more information about the mentors in the sessions.

**Comment by code-for-archives** [comment]
I have started learning about Python and found that Python for Everybody is pretty helpful. Not up to being able to batch rename files yet but getting a feel for how to use Python.

**Comment by WMRG1057** [comment]
Was there suppose to be a meeting today March 27th

**Reply by barsness13** [reply]
A few of us think that daylight savings time might be the culprit; it happens on 31 March in England but has already happened in many other locations (and doesn't happen at all in others).

**Reply by WMRG1057** [reply]
that could be a problem
if I can I'll check back at 10 EDT
Have another one at 11

**Comment by asegal20902** [comment]
Yes I am going to try to get on in a minute 


Sent from Yahoo Mail for iPhone


On Wednesday, March 27, 2024, 9:14 AM, WMRG1057 ***@***.***> wrote:




Was there suppose to be a meeting today March 27th

**Reply by WMRG1057** [reply]
https://us02web.zoom.us/postattendee?mn=SMbum5Xbzvq3hYs4uBPqtCARfCHSrreJoufl.OIr46BndBw8t2-6A

this is the link I have

**Reply by WMRG1057** [reply]
correction

[https://us02web.zoom.us/j/85426211216?pwd=M2gzbU95a2JHL3RTbkNZWWlwTkVQQT09](https://www.google.com/url?q=https://us02web.zoom.us/j/85426211216?pwd%3DM2gzbU95a2JHL3RTbkNZWWlwTkVQQT09&sa=D&source=calendar&usd=2&usg=AOvVaw3OBJtLBeFDk6yB51zFw3-3)
